platform-operator/helm-pod/main.go

```diff
@@ -380,6 +383,93 @@ func deleteHelmRelease(helmreleaseNS, helmrelease string) bool {
 	return ok
 }
 
+func updateCRDInstances(request *restful.Request, response *restful.Response) {
+	fmt.Printf("-- Inside updateCRDInstances...\n")
+
+	kind := request.QueryParameter("kind")
+	group := request.QueryParameter("group")
+	version := request.QueryParameter("version")
+	plural := request.QueryParameter("plural")
+	namespace := request.QueryParameter("namespace")
+	chartURL := request.QueryParameter("chartURL")
+	chartName := request.QueryParameter("chartName")
+	crName := request.QueryParameter("instance")
+
+	fmt.Printf("Kind:%s\n", kind)
+	fmt.Printf("Group:%s\n", group)
+	fmt.Printf("Version:%s\n", version)
+	fmt.Printf("Plural:%s\n", plural)
+	fmt.Printf("Namespace:%s\n", namespace)
+	fmt.Printf("ChartURL:%s\n", chartURL)
+	fmt.Printf("ChartName:%s\n", chartName)
+	fmt.Printf("CRName:%s\n", crName)
+
+	if _, errF := os.Stat("/" + chartName); os.IsNotExist(errF) {
+		fmt.Printf("Error: chart '%s' not installed", chartName)
+	}
```

Comment:
We can return at this point if the chart is not installed.

errFString := convert errF to String
response.Write([]byte(errFString))

Reply:
Done.